installer/windows/build_installer.py

```diff
@@ -463,13 +463,14 @@ def build_installer():
     except err:
         print(err)
         sys.exit(1)
+    print(f"Installer built successfully: massastation_{VERSION}_amd64.msi")
```

Comment:
debug ?

Reply:
Looks like so
But I think it's not a bad addition and so we can keep it